src/ChosenOne.js

```diff
@@ -1,19 +1,44 @@
-// import React, { Component} from 'react';
+import React from 'react';
 import './ChosenOne.css';
+import PropTypes from 'prop-types';
 
 const ChosenOne = (props) => {
 
+// let src= videoSite(props.trailer[0].site)
+
 return (
   <div 
     className='single-movie'
     >
     <h1>{props.details.title}</h1>
+    <div className="video-responsive">
+      <iframe
+        width="800"
+        height="480"
+        src={props.trailer.length &&  `https://www.youtube.com/embed/${props.trailer[0].key}`}
+        frameBorder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowFullScreen
+        title="Embedded youtube"
+      />
+    </div>
     <h2>{props.details.overview}</h2>
     <img className='movie-pic'src={props.details.backdrop_path} alt='movie poster' />
   </div>
 )
 
 }
+
+// Pseudo-code for videos from vimeo - not sure if we will have time to do that
```

Comment:
This will definitely be helpful when we come back and wonder what we were thinking about this :)